### DISCUSSION TITLE
Opinion: Lead w/PostCSS (Just In Time) Props

### DISCUSSION THREAD

**Discussion by roblevintennis** [argyleink/open-props#113] [Ideas]
Maybe I'm in the minority here, but I think the power of OP is actually its partnership with this value prop in the JIT library (which I've yet to actually try but totally get the intent and like it):

> This lets you provide a huge pool of properties for development and design, and rather than try and purge unused variables, only adds what was used.

For me, this is pretty huge selling point. I'd say if I can do this (prune all but those I've used), and somehow control the values themselves when I want to, I'd be like 💯

**Comment by argyleink** [comment]
It's definitely a more advanced use case, would rather lead with the CDN, no build steps, usage. Especially since the main bundle is so tiny. The folks concerned with delivering unused props can find these more advanced strategies to keep their bundles minimal. 

I agree jit-props and open-props make for a very nice setup, but can't expect everyone to setup postcss and node just to use the props. Letting folks layer in complexity seems like a more accessible option?

**Reply by roblevintennis** [reply]
Yeah, I'm definitely not the typical user and what you say makes sense.
